docs/ftldns/blockingmode.md

```diff
@@ -42,52 +47,50 @@ doubleclick.net.        2       IN      A       192.168.2.11
 ```
 
 ##### Advantage
-- Shows blocking page from which blocked webpages can be whitelisted
+- Shows blocking page from which blocked domains can be whitelisted
 - Serves IPv4-only replies and hence mitigates issues with rotating IPv6 prefixes
 
 ##### Disadvantages
 - Requires a webserver to run on your Pi-hole
 - May cause time-outs for HTTPS content even with properly configured firewall rules
```

Comment:
Even with properly configured firewall rules?

Reply:
Yes, we have seen several reports where even after really fixing anything with firewalls, etc. it still didn't work. We found that in all these cases very locked down ISP routers (partially from Google) have been used so they might want to do some "protection" for their users...